appinfo/info.xml

```diff
@@ -48,7 +48,7 @@
 	<repository type="git">https://github.com/nextcloud/gallery.git</repository>
 	<dependencies>
 		<php min-version="5.6" max-version="7"/>
```

Comment:
@oparoz does this work with php 7.1 and upcoming 7.2?

Reply:
It should support the same versions as nextcloud 14 because its shipped

Reply:
No problem to make it compatible with 7.2 for 14

Reply:
14 is 7+, no?`If that's the case, let's update min-max to 7-7.2

Reply:
14 is 7.0 - 7.2 for now yes